### PR TITLE
pkg add: fcitx5-rime

### PR DESCRIPTION
**Pull request by eatradish**
https://gitlab.com/fcitx/fcitx5-rime

### CODE REVIEW

File: extra-i18n/fcitx5-rime/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,4 @@
+PKGNAME=fcitx5-rime
+PKGDES="Fcitx5 wrapper for librime"
+PKGDEP="fcitx5 boost librime opencc brise"
```
Comment (liushuyu) [inline extra-i18n/fcitx5-rime/autobuild/defines]: Explain why `opencc` is required.

Comment (eatradish) [reply extra-i18n/fcitx5-rime/autobuild/defines]: > Explain why `opencc` is required.

`brise` need `opencc`?....ok, actually can remove it...

Comment (eatradish) [reply extra-i18n/fcitx5-rime/autobuild/defines]: and maybe need `curl` used for download cloud pinyin etc...

Comment (liushuyu) [reply extra-i18n/fcitx5-rime/autobuild/defines]: >brise need opencc?....ok, actually can remove it...

 No, `brise` does not require `opencc`. It is `librime` whose dependencies include `opencc`.

Comment (liushuyu) [inline extra-i18n/fcitx5-rime/autobuild/defines]:
```suggestion
PKGDEP="fcitx5 boost librime brise"
```

**Review by liushuyu — CHANGES_REQUESTED** [review]
Remove redundant deps

**Pull request merged**